pbxproj-lang/src/main/java/org/greencodeinitiative/creedengo/ios/pbxproj/checks/sobriety/DisabledDarkModeCheck.java

```diff
@@ -15,16 +15,16 @@
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-package io.ecocode.ios.pbxproj.checks.sobriety;
+package org.greencodeinitiative.creedengo.ios.pbxproj.checks.sobriety;
 
 import org.antlr.v4.runtime.tree.ParseTree;
 
-import io.ecocode.ios.pbxproj.PbxprojRuleCheck;
-import io.ecocode.ios.pbxproj.antlr.generated.PbxprojParser;
+import org.greencodeinitiative.creedengo.ios.pbxproj.PbxprojRuleCheck;
+import org.greencodeinitiative.creedengo.ios.pbxproj.antlr.generated.PbxprojParser;
 
 import org.sonar.check.Rule;
 
-@Rule(key = "EC547")
+@Rule(key = "GCI547")
```

Comment:
the "@deprecatedRule" is missing on all rules.
please check creedengo-python for an example

Reply:
done !